fastapi-google-stt/app/services/openai_vector_store.py

```
@@ -27,15 +27,15 @@
     doc = Document(page_content=text, metadata={
         "script_id": script_id,
         "lang": language_code,
         "source": "stt",
         "created_at":now
     })    # 문서 객체로 변환
+    vectordb = init_vectordb()
     vectordb.add_documents([doc])                      # 벡터 DB에 추가
-
+    
 # 유사 텍스트 검색
 def search_text(query: str, top_k: int = 3, meta_data: dict = {}):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)